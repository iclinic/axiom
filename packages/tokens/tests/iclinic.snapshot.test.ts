import { expect, test } from "vitest";

import { iclinic } from "../dist/ts/iClinic/theme";

test("iclinic theme tokens should match snapshot", () => {
  try {
    expect(iclinic).toMatchSnapshot();
  } catch (err) {
    console.error(
      '‚ùå Ops! You are changing final iClinic theme tokens, if you\'re really sure about your changes please update snapshot through command "npm run test:update" and after review commit your changes'
    );
    throw err;
  }
});
