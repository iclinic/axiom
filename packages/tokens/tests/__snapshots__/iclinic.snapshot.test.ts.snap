// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`iclinic theme tokens should match snapshot 1`] = `
{
  "border": {
    "none": "0px",
    "sm": "2px",
    "xs": "1px",
  },
  "color": {
    "backdrop": "#0000006e",
    "bg": {
      "base": {
        "1": "#fff",
        "2": "#f7f7f7",
        "3": "#e6e6e6",
      },
    },
    "border": {
      "accent": "#0000003b",
      "activated": "#0b63cb",
      "disabled": "#0000002b",
      "feedback": {
        "alert": "#e02424",
        "info": "#046eec",
        "success": "#0e8610",
        "warning": "#c76a00",
      },
      "subtle": "#0000001a",
    },
    "brand": {
      "backdrop": "#0000006e",
      "bg": {
        "sabe": {
          "1": "#0b63cb",
          "2": "#0857b4",
          "3": "#024aa2",
        },
      },
      "border": {
        "accent": "#ffffffa3",
        "activated": "#f0f7ff",
        "disabled": "#ffffff78",
        "feedback": {
          "alert": "#fac7c7",
          "info": "#b8d9ff",
          "success": "#bcdcbc",
          "warning": "#ffcd6b",
        },
        "subtle": "#ffffff85",
      },
      "divider": "#ffffff5c",
      "focus": {
        "default": "#f0f7ff",
        "inset": "#046eec0f",
      },
      "highlight": {
        "blue": {
          "1": "#99c8ff",
          "2": "#b8d9ff",
          "3": "#d6e9ff",
        },
        "orange": {
          "1": "#fcb06e",
          "2": "#fdc99b",
          "3": "#fee1c8",
        },
        "purple": {
          "1": "#d2b8f4",
          "2": "#e0ccfa",
          "3": "#eee2fd",
        },
        "red": {
          "1": "#fdafaf",
          "2": "#fac7c7",
          "3": "#ffe0e0",
        },
        "ruby": {
          "1": "#f1b1ce",
          "2": "#f5c7dc",
          "3": "#f9dde9",
        },
        "teal": {
          "1": "#68d4bf",
          "2": "#8de2d1",
          "3": "#bcf0e6",
        },
      },
      "icon": {
        "activated": "#f0f7ff",
        "default": "#ffffffd6",
        "disabled": "#ffffff94",
        "feedback": {
          "alert": "#fac7c7",
          "info": "#b8d9ff",
          "success": "#bcdcbc",
          "warning": "#ffcd6b",
        },
        "hovered": "#ffffffc4",
        "onInteractive": {
          "accent": "#003e8a",
          "alert": {
            "accent": "#890101",
            "subtle": "#fef6f6",
          },
          "subtle": "#f0f7ff",
        },
        "pressed": "#ffffffb5",
      },
      "interactive": {
        "accent": {
          "default": "#f0f7ff",
          "hovered": "#d6e9ff",
          "pressed": "#b8d9ff",
        },
        "alert": {
          "accent": {
            "default": "#ffe0e0",
            "hovered": "#fac7c7",
            "pressed": "#fe9494",
          },
          "subtle": {
            "default": "#f63737c9",
            "hovered": "#cf0303",
            "pressed": "#b80000",
          },
        },
        "disabled": "#ffffff5c",
        "subtle": {
          "default": "#0000003b",
          "hovered": "#0000004d",
          "pressed": "#0000005e",
        },
      },
      "link": {
        "default": "#f0f7ff",
        "visited": "#e0ccfa",
      },
      "skeleton": "#ffffffa3",
      "surface": {
        "accent": {
          "default": "#0000001a",
          "hovered": "#0000002b",
          "pressed": "#0000003b",
        },
        "disabled": "#0000002b",
        "feedback": {
          "alert": {
            "accent": "#cf0303",
            "subtle": "#fd5d5d0f",
          },
          "info": {
            "accent": "#0000003b",
            "subtle": "#046eec0f",
          },
          "success": {
            "accent": "#0c790e",
            "subtle": "#37953912",
          },
          "warning": {
            "accent": "#9e5400",
            "subtle": "#db790014",
          },
        },
        "selected": {
          "default": "#0000003b",
          "hovered": "#0000004d",
          "pressed": "#0000004d",
        },
        "subtle": {
          "default": "#0000000a",
          "hovered": "#0000001a",
          "pressed": "#0000002b",
        },
      },
      "text": {
        "feedback": {
          "alert": "#ffe0e0",
          "info": "#d6e9ff",
          "success": "#daecda",
          "warning": "#ffe3ad",
        },
        "meld": "#ffffff69",
        "onInteractive": {
          "accent": "#003e8a",
          "alert": {
            "accent": "#890101",
            "subtle": "#fef6f6",
          },
          "disabled": "#ffffff94",
          "subtle": "#f0f7ff",
        },
        "pure": "#ffffffe3",
        "smooth": "#ffffffd6",
        "strong": "#fff",
      },
    },
    "branding": {
      "10": "#001f3d",
      "100": "#3894ff",
      "110": "#57a4ff",
      "120": "#75b5ff",
      "130": "#99c8ff",
      "140": "#b8d9ff",
      "150": "#d6e9ff",
      "160": "#f0f7ff",
      "20": "#002a52",
      "30": "#07356e",
      "40": "#003e8a",
      "50": "#024aa2",
      "60": "#0857b4",
      "70": "#0b63cb",
      "80": "#046eec",
      "90": "#0a7bff",
      "alfa": {
        "100": "#046eecba",
        "110": "#046eec9e",
        "120": "#046eec82",
        "130": "#046eec61",
        "140": "#046eec45",
        "150": "#046eec26",
        "160": "#046eec0f",
        "90": "#046eeccf",
      },
    },
    "divider": "#0000001a",
    "focus": {
      "default": "#0b63cb",
      "inset": "#ffffffd6",
    },
    "highlight": {
      "blue": {
        "1": "#3894ff",
        "2": "#046eec",
        "3": "#0857b4",
      },
      "orange": {
        "1": "#eb7005",
        "2": "#b95804",
        "3": "#914503",
      },
      "purple": {
        "1": "#ab7ce9",
        "2": "#8f51e1",
        "3": "#732dd2",
      },
      "red": {
        "1": "#fd5d5d",
        "2": "#e02424",
        "3": "#b80000",
      },
      "ruby": {
        "1": "#e368a0",
        "2": "#d82777",
        "3": "#a81f5d",
      },
      "teal": {
        "1": "#12a588",
        "2": "#0f856d",
        "3": "#0b6553",
      },
    },
    "icon": {
      "activated": "#0b63cb",
      "default": "#0000007d",
      "disabled": "#0000003b",
      "feedback": {
        "alert": "#f63737",
        "info": "#0a7bff",
        "success": "#379539",
        "warning": "#c76a00",
      },
      "hovered": "#0000008c",
      "onInteractive": {
        "accent": "#f0f7ff",
        "alert": {
          "accent": "#fef6f6",
          "subtle": "#890101",
        },
        "subtle": "#003e8a",
      },
      "pressed": "#00000099",
    },
    "interactive": {
      "accent": {
        "default": "#0b63cb",
        "hovered": "#0857b4",
        "pressed": "#024aa2",
      },
      "alert": {
        "accent": {
          "default": "#cf0303",
          "hovered": "#b80000",
          "pressed": "#a00303",
        },
        "subtle": {
          "default": "#fef6f6",
          "hovered": "#ffe0e0",
          "pressed": "#fac7c7",
        },
      },
      "disabled": "#0000001a",
      "subtle": {
        "default": "#f0f7ff",
        "hovered": "#d6e9ff",
        "pressed": "#b8d9ff",
      },
    },
    "inverse": {
      "backdrop": "#000000b3",
      "bg": {
        "base": {
          "1": "#1f1f1f",
          "2": "#292929",
          "3": "#363636",
        },
      },
      "border": {
        "accent": "#ffffff94",
        "activated": "#f0f7ff",
        "disable": "#ffffff2b",
        "feedback": {
          "alert": "#f63737",
          "info": "#0a7bff",
          "success": "#379539",
          "warning": "#c76a00",
        },
        "subtle": "#ffffff45",
      },
      "divider": "#ffffff45",
      "focus": {
        "default": "#f0f7ff",
        "inset": "#0000000a",
      },
      "highlight": {
        "blue": {
          "1": "#046eec",
          "2": "#3894ff",
          "3": "#75b5ff",
        },
        "orange": {
          "1": "#b95804",
          "2": "#eb7005",
          "3": "#fb9b46",
        },
        "purple": {
          "1": "#8f51e1",
          "2": "#ab7ce9",
          "3": "#c4a3f0",
        },
        "red": {
          "1": "#e02424",
          "2": "#fd5d5d",
          "3": "#fe9494",
        },
        "ruby": {
          "1": "#d82777",
          "2": "#e368a0",
          "3": "#ec98be",
        },
        "teal": {
          "1": "#0f856d",
          "2": "#12a588",
          "3": "#31c4a6",
        },
      },
      "icon": {
        "activated": "#f0f7ff",
        "default": "#ffffffd6",
        "disabled": "#ffffff78",
        "feedback": {
          "alert": "#fd5d5d",
          "info": "#3894ff",
          "success": "#56a457",
          "warning": "#db7900",
        },
        "hovered": "#ffffffc4",
        "onInteractive": {
          "accent": "#003e8a",
          "alert": {
            "accent": "#890101",
            "subtle": "#fef6f6",
          },
          "subtle": "#f0f7ff",
        },
        "pressed": "#ffffffb5",
      },
      "interactive": {
        "accent": {
          "default": "#f0f7ff",
          "hovered": "#d6e9ff",
          "pressed": "#b8d9ff",
        },
        "alert": {
          "accent": {
            "default": "#ffe0e0",
            "hovered": "#fac7c7",
            "pressed": "#fdafaf",
          },
          "subtle": {
            "default": "#cf0303",
            "hoverd": "#b80000",
            "pressed": "#a00303",
          },
        },
        "disabled": "#ffffff5c",
        "subtle": {
          "default": "#0b63cb",
          "hovered": "#0857b4",
          "pressed": "#024aa2",
        },
      },
      "link": {
        "default": "#f0f7ff",
        "visited": "#8f51e1",
      },
      "skeleton": "#ffffff45",
      "surface": {
        "accent": {
          "default": "#ffffff1c",
          "hovered": "#ffffff2b",
          "pressed": "#ffffff36",
        },
        "disabled": "#ffffff2b",
        "feedback": {
          "alert": {
            "accent": "#cf0303",
            "subtle": "#fd5d5d0f",
          },
          "info": {
            "accent": "#0b63cb",
            "subtle": "#046eec0f",
          },
          "success": {
            "accent": "#0c790e",
            "subtle": "#37953912",
          },
          "warning": {
            "accent": "#9e5400",
            "subtle": "#db790014",
          },
        },
        "selected": {
          "default": "#046eec82",
          "hovered": "#046eec9e",
          "pressed": "#046eecba",
        },
        "subtle": {
          "default": "#0000000a",
          "hovered": "#ffffff1c",
          "pressed": "#ffffff2b",
        },
      },
      "text": {
        "feedback": {
          "alert": "#fe9494",
          "info": "#75b5ff",
          "success": "#88bf89",
          "warning": "#f59f00",
        },
        "meld": "#ffffff78",
        "onInteractive": {
          "accent": "#003e8a",
          "alert": {
            "accent": "#890101",
            "subtle": "#fef6f6",
          },
          "disabled": "#ffffff78",
          "subtle": "#f0f7ff",
        },
        "pure": "#ffffffd6",
        "smooth": "#ffffffb5",
        "strong": "#ffffffe3",
      },
    },
    "link": {
      "default": "#0b63cb",
      "visited": "#641ec2",
    },
    "skeleton": "#0000001a",
    "surface": {
      "accent": {
        "default": "#0000000a",
        "hovered": "#0000001a",
        "pressed": "#0000002b",
      },
      "disabled": "#0000001a",
      "feedback": {
        "alert": {
          "accent": "#cf0303",
          "subtle": "#fef6f6",
        },
        "info": {
          "accent": "#0b63cb",
          "subtle": "#f0f7ff",
        },
        "success": {
          "accent": "#0c790e",
          "subtle": "#f2f8f2",
        },
        "warning": {
          "accent": "#9e5400",
          "subtle": "#fff5e0",
        },
      },
      "selected": {
        "default": "#046eec0f",
        "hovered": "#046eec26",
        "pressed": "#046eec45",
      },
      "subtle": {
        "default": "#ffffffe3",
        "hovered": "#0000000a",
        "pressed": "#0000001a",
      },
    },
    "text": {
      "feedback": {
        "alert": "#b80000",
        "info": "#0857b4",
        "success": "#076908",
        "warning": "#8a4900",
      },
      "meld": "#0000003b",
      "onInteractive": {
        "accent": "#f0f7ff",
        "alert": {
          "accent": "#fef6f6",
          "subtle": "#890101",
        },
        "disabled": "#0000004d",
        "subtle": "#003e8a",
      },
      "pure": "#000000bd",
      "smooth": "#0000008c",
      "strong": "#000000d6",
    },
  },
  "font": {
    "bold": 700,
    "family": "Roboto",
    "medium": 500,
    "regular": 400,
  },
  "leading": {
    "lg": "142%",
    "md": "132%",
    "sm": "128%",
    "xl": "150%",
    "xs": "116%",
    "xxs": "100%",
  },
  "opacity": {
    "1": 0.04,
    "10": 0.96,
    "2": 0.08,
    "3": 0.12,
    "4": 0.16,
    "5": 0.24,
    "6": 0.32,
    "7": 0.48,
    "8": 0.64,
    "9": 0.8,
  },
  "rounded": {
    "full": "10000px",
    "lg": "16px",
    "md": "8px",
    "none": "0px",
    "sm": "4px",
    "xl": "24px",
    "xxl": "32px",
  },
  "shadow": {
    "1": "0px 1px 2px 0px rgba(0, 0, 0, 0.04)",
    "2": "0px 1px 2px 0px rgba(0, 0, 0, 0.04), 0px 5px 4px -4px rgba(0, 0, 0, 0.04)",
    "3": "0px 2px 2px 0px rgba(0, 0, 0, 0.04), 0px 0px 2px 0px rgba(0, 0, 0, 0.04)",
    "4": "0px 12px 12px -8px rgba(0, 0, 0, 0.1), 0px 0px 4px 0px rgba(0, 0, 0, 0.04)",
    "5": "0px 32px 40px -16px rgba(0, 0, 0, 0.1), 0px 2px 6px 0px rgba(0, 0, 0, 0.04)",
    "6": "0px 24px 48px -8px rgba(0, 0, 0, 0.17), 0px 48px 64px -24px rgba(0, 0, 0, 0.1)",
  },
  "shadowBrand": {
    "1": "0px 1px 2px 0px rgba(4, 110, 236, 0.06)",
    "2": "0px 1px 2px 0px rgba(4, 110, 236, 0.06), 0px 5px 4px -4px rgba(4, 110, 236, 0.06)",
    "3": "0px 2px 2px 0px rgba(4, 110, 236, 0.06), 0px 0px 2px 0px rgba(4, 110, 236, 0.06)",
    "4": "0px 12px 12px -8px rgba(4, 110, 236, 0.15), 0px 0px 4px 0px rgba(4, 110, 236, 0.06)",
    "5": "0px 32px 40px -16px rgba(4, 110, 236, 0.15), 0px 2px 6px 0px rgba(4, 110, 236, 0.06)",
    "6": "0px 24px 48px -8px rgba(4, 110, 236, 0.15), 0px 48px 64px -24px rgba(4, 110, 236, 0.15)",
  },
  "space": {
    "1": "0.25rem",
    "10": "2.5rem",
    "12": "3rem",
    "14": "3.5rem",
    "16": "4rem",
    "2": "0.5rem",
    "20": "5rem",
    "24": "6rem",
    "3": "0.75rem",
    "32": "8rem",
    "4": "1rem",
    "40": "10rem",
    "5": "1.25rem",
    "56": "14rem",
    "6": "1.5rem",
    "72": "18rem",
    "8": "2rem",
  },
  "text": {
    "lg": "1.75rem",
    "md": "1.5rem",
    "sm": "1.125rem",
    "xl": "2.25rem",
    "xs": "1rem",
    "xxl": "2.5rem",
    "xxs": "0.875rem",
    "xxxs": "0.75rem",
  },
  "tracking": {
    "md": "1px",
    "sm": "0.72px",
  },
}
`;
